Dashboard/navigation.py

```diff
@@ -0,0 +1,47 @@
+"""Controls the sidebar for streamlit"""
+
+from time import sleep
+
+import streamlit as st
+from streamlit.runtime.scriptrunner import get_script_run_ctx
+from streamlit.source_util import get_pages
+
+
+def get_current_page_name() -> str | None:
+    """Gets the name of the page"""
+    ctx = get_script_run_ctx()
+    if ctx is None:
+        raise RuntimeError("Couldn't get script context")
+
+    pages = get_pages("")
+
+    return pages[ctx.page_script_hash].get("page_name")
+
+
+def make_sidebar() -> None:
+    """Creates sidebar"""
+    with st.sidebar:
+        st.title("💎 Sales tracker")
+        st.write("")
+        st.write("")
+
+        if st.session_state.get("logged_in", False):
+            st.page_link("pages/price.py",
+                         label="Price Tracker", icon="💵")
+
+            st.write("")
+            st.write("")
```

Comment:
Could with work instead?

```suggestion
            st.write("\n\n")
```

Reply:
Doesn't work